### ISSUE TITLE
Bind address both in jboss.properties and command line

### ISSUE THREAD

**Issue by rgarrigue** [biemond/biemond-wildfly#162]
Hi again

There's a WILDFLY_BIND in /etc/wildfly/wildfly.conf which end up in service start

    ExecStart=/bin/bash -x /opt/wf10/bin/launch.sh $WILDFLY_MODE $WILDFLY_CONFIG $WILDFLY_BIND

But I also have this in jboss.properties

```
[root@testcentos ~]# cat /opt/wf10/jboss.properties 
jboss.ajp.port = 8009
jboss.bind.address = 0.0.0.0
jboss.bind.address.management = 127.0.0.1
jboss.http.port = 8080
jboss.https.port = 8443
jboss.management.http.port = 9990
jboss.management.https.port = 9993
```

I dunno which one is taken into account, and I'ld prefer not having both

**Comment by jairojunior** [comment]
As I said in the other issue, I brought this from the official Wildfly distribution cause I'd rather use bundled init scripts then provide my own.

But since I'll be fully managing these scripts due to #161. I'll remove `-b $3` .

Thanks for being an early adopter. :smile: 

There is a lot of tests for this version, but some details still pass unnoticed.

**Comment by jairojunior** [comment]
Oh, and I think Wildfly will use `-b` bind instead of properties.

**Comment by jairojunior** [comment]
Fixed in v1.0.1. Thanks again.
